pkg/analyze/kube_resource.go

```diff
@@ -108,15 +109,28 @@ func FindResource(kind string, clusterScoped bool, namespace string, name string
 }
 
 func (a *AnalyzeClusterResource) analyzeResource(analyzer *troubleshootv1beta2.ClusterResource, getFileContents getCollectedFileContents) (*AnalyzeResult, error) {
-
 	selected, err := FindResource(analyzer.Kind, analyzer.ClusterScoped, analyzer.Namespace, analyzer.Name, getFileContents)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to find resource")
+		klog.Errorf("failed to find resource: %v", err)
+		return &AnalyzeResult{
+			Title:   a.Title(),
+			IconKey: "kubernetes_text_analyze",
+			IconURI: "https://troubleshoot.sh/images/analyzer-icons/text-analyze.svg",
+			IsFail:  true,
+			Message: "resource does not exist",
+		}, nil
 	}
 
 	actual, err := iutils.GetAtPath(selected, analyzer.YamlPath)
 	if err != nil {
-		return nil, errors.Wrapf(err, "failed to get object at path: %s", analyzer.YamlPath)
+		klog.Errorf("invalid yaml path: %s for kind: %s: %v", analyzer.YamlPath, analyzer.Kind, err)
```

Comment:
The message is clearer now.

I was coming from the POV of the cli where is the spec is wrong i.e invalid regex, jsonpath etc, the cli would return an error code as per https://troubleshoot.sh/docs/preflight/exit-codes/.

I'm not so fussy but later on when we start [validating specs](https://github.com/replicatedhq/troubleshoot/issues/871), this bit will change.